testutils/src/main/scala/za/co/absa/enceladus/testutils/RandomDataGenerator.scala

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils
+
+import java.io.{BufferedWriter, File, FileWriter}
+
+import org.joda.time.DateTime
+import org.joda.time.format.{DateTimeFormat, DateTimeFormatter}
+
+import scala.util.Random
+
+object RandomDataGenerator {
+
+  // Set DateTime format to be used
+  val dtf: DateTimeFormatter = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss")
```

Comment:
Is it expected this to be used outside of the object? If yes, perhaps a better name would be suitable, if not make private.